### PR TITLE
CONTRIBUTING.md and GitHub issue template

### PR DESCRIPTION
**Pull request by mstenta**
See discussion in https://github.com/farmOS/farmOS/issues/374

Corresponding PR in farmOS.org: https://github.com/farmOS/farmOS.org/pull/102

### CODE REVIEW

**Comment by mstenta** [comment]
I updated https://www.drupal.org/project/farm/issues/3162606 to include these new files, so they will be merged into 2.x at the same time as other common files.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+**Questions**, **support requests**, and **general discussions** should be in
+the [farmOS Forum](https://farmOS.discourse.group).
+
+**Bug reports** can be posted to the
+[farmOS issue queue on GitHub](https://github.com/farmOS/farmOS/issues).
+Issues in this queue will be triaged and moved or copied into the appropriate
+issue queues for the code responsible for them.
+
+**Feature requests** should start in the forum, most of the time. Developers
```
Comment (symbioquine) [inline CONTRIBUTING.md]: Probably don't need the comma here.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+modular and API-driven approach to development allows for experimentation and
+innovation in the "contrib" space. If core farmOS features are built, issues
+for each specific piece can be created in the appropriate project issue queues
+and added to their respective roadmaps.
+
+**Core development** issues and roadmaps are managed on Drupal.org in the
+[farmOS Project](https://drupal.org/project/farm) issue queue:
+[https://drupal.org/project/issues/farm](https://drupal.org/project/issues/farm)
+
+## Live discussions
```
Comment (symbioquine) [inline CONTRIBUTING.md]: Inconsistent title case.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+necessary to talk to someone in real time.
+
+The farmOS community hosts a monthly call on the second Wednesday of the month.
+Often these calls are free-form "office hours" to get support or discuss ideas,
+and sometimes they have a specific agenda or focus. The link and call schedule
+are available on farmOS.org:
+
+**[farmOS Monthly Call](https://farmOS.org/community/monthly-call)**
+
+We also have a farmOS chat room, which is available on both Matrix.org and IRC.
```
Comment (symbioquine) [inline CONTRIBUTING.md]: Most of the paragraphs are written in the third-person, but this one and the one about translations use "we". Might make sense to be consistent...

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+issue queues for the code responsible for them.
+
+**Feature requests** should start in the forum, most of the time. Developers
+and other community members can discuss the idea and decide where it makes
+sense to build it. In many cases, new features can be built as standalone
+modules or apps, and don't need to be part of the farmOS code itself. This
+modular and API-driven approach to development allows for experimentation and
+innovation in the "contrib" space. If core farmOS features are built, issues
+for each specific piece can be created in the appropriate project issue queues
+and added to their respective roadmaps.
```
Comment (symbioquine) [inline CONTRIBUTING.md]: > If core farmOS features are built, issues for each specific piece can be created in the appropriate project issue queues
and added to their respective roadmaps.

Seems a little unclear why the assignment of issues is predicated on "core farmOS features [being] built". Shouldn't this say something more like; "**If the feature request requires core farmOS feature(s) to be built**, issues for for each specific piece can be created..."?

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+## Use farmOS
+
+First and foremost: use farmOS.
+
+Building a critical mass of farmOS users is the best way for the project to
+grow and thrive. If you would like to improve your record keeping habits, or
+just want a place to put everything so you can access it easily, farmOS can
+help. If you have a social media presence, or if you know farmers - help tell
+the world about farmOS!
```
Comment (symbioquine) [inline CONTRIBUTING.md]: Maybe this should be phrased to "normalize knowing farmers"; e.g. "Tell the farmers you know about farmOS and - if you have a social media presence - help tell the world about farmOS!"

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+There are many ways to get involved in the farmOS community. You don't need to
+be a programmer. All are welcome to take part in the process.
+
+## Use farmOS
+
+First and foremost: use farmOS.
+
+Building a critical mass of farmOS users is the best way for the project to
+grow and thrive. If you would like to improve your record keeping habits, or
+just want a place to put everything so you can access it easily, farmOS can
```
Comment (symbioquine) [inline CONTRIBUTING.md]: Maybe this should be a tiny bit more specific about the use-case niche for farmOS; "If you would like to improve your **farming/gardening/agricultural** record keeping habits, or just want a place to put everything so you can access it easily, farmOS can help."

**Comment by mstenta** [comment]
Thanks @symbioquine - I made all the changes you recommended and force-pushed.

**Pull request merged**